galata/test/jupyterlab/terminal.test.ts

```diff
@@ -84,3 +84,16 @@ test.describe('Terminal', () => {
     });
   });
 });
+
+test('Terminal should open in Launcher cwd', async ({ page, tmpPath }) => {
+  await page.waitForSelector(`.jp-Launcher-cwd > h3:has-text("${tmpPath}")`);
+
+  await page.locator('[role="main"] >> p:has-text("Terminal")').click();
+
+  const terminal = page.locator(TERMINAL_SELECTOR);
+  await terminal.waitFor();
+
+  await page.keyboard.type('pwd');
```

Comment:
What about using `basename "$PWD"` instead to just get the name of the directory (and not the full path)? @fcollonval

Reply:
It would indeed be good to test that.